products/firefox_android/how_to_localize.md

```diff
@@ -0,0 +1,53 @@
+
+Checking your team page and dashboard regularly should be something you do from the very beginning when you start localizing a project, so that it becomes a habit. This helps reduce the chance of breaking the builds in the future, once your locale is actually shipping, and generally helps catch errors earlier on. It also gives you an overview of your locale’s health and status, and overall ensures you’re shipping a good quality localization.
+
+## Localize the Firefox for Android folder
+
+* [Pontoon](https://pontoon.mozilla.org/firefox-for-android-aurora/): choose your locale code from there.
+* [Pootle](https://mozilla.locamotion.org/projects/mobile/ ): choose your locale code from there.
+
+Once you’ve finished covering this folder, please note you still have strings to translate. See below for more instructions.
+
+## Localize the shared Firefox strings
+
+In order to reach 100% completion on the Firefox for Android l10n project, you will have to localize the shared Firefox files (so, the ones common to Firefox Desktop and Android) as well. These are found in the Firefox Aurora folder:
+* [Pontoon](https://pontoon.mozilla.org/projects/firefox-aurora/): choose your locale code from there. Localize all file paths that start with dom, netwerk, security, services and toolkit.
+* [Pootle](https://mozilla.locamotion.org/projects/firefox/): choose your locale code from there. In the list you now see, localize the files found in "androidshared" folder.
+
+Once you’ve reached 100% completion for the shared strings: let the Fennec project manager know (currently **delphine** at **mozilla** dot **com**). A bug will be filed to add your locale to the single-locale builds.
+
+These builds are available here: https://ftp.mozilla.org/pub/mobile/nightly/latest-mozilla-aurora-android-api-15-l10n/.
+This means that you will be able to download a build on your Android device to test your locale. Please note that at this point, your locale still isn’t shipping on official Firefox for Android builds. That will come later.
+
+## Considerations for brand new locales that have no Desktop version yet
+
+If you have not already launched Firefox Desktop in your language, then you are probably missing some important pieces, such as search engines.
+
+Bugs will need to be filed by l10n-drivers, and done by following filing out the form here: https://l10n.mozilla.org/bugs/new-locale
+
+Please read this [Firefox for Android "setup_searchplugins" section](/products/firefox_desktop/setup_searchplugins.md) for more details on how to add search plugins.
+
+Once you have finished localization work on Android and your locale is up on single-locale builds, you may now move to [the testing phase](/products/firefox_android/testing_android.md).
```

Comment:
This appears broken but I assume it's going to be the same shortcut in the new folder, so won't matter once the other PR is merged. Not sure :P